CHANGES.rst

```diff
@@ -1,6 +1,47 @@
 Release Notes
 =============
 
+2.1.105
+-------
+
+This is a fix release which addresses issues related to build time work_dir creation,
+virtualenv and sh_boot support, as well as Windows compatibility. Also includes various CI fixes and improvements.
+
+In the unlikely event of a UUID collision in atomic workdir creation,
+pex could overwrite an existing directory and cause a corrupt state.
+When building a shell bootable ``--sh-boot`` pex the ``--runtime-pex-root``
+was not always respected based on the condition of the build environment,
+and the value of the PEX_ROOT.
```

Comment:
Per the nixes below I'd also adjust this to only refer to publicly visible things. I consider Windows not real until its real. We still definitely do not support Windows at all. Itall or nothing.